jsonschema-hyperschema.xml

```diff
@@ -387,38 +387,87 @@
                 <t>
-                    Data input functionality can be defined by use of the
-                    <xref target="submissionSchema">"submissionSchema"</xref> and
-                    <xref target="submissionEncType">"submissionEncType"</xref> keywords,
-                    which supply a description of data to send to the target resource for processing.
-                    Functionality equivalent to dynamic URI generation is available through
-                    the <xref target="href">"href"</xref> template and
-                    <xref target="hrefSchema">"hrefSchema"</xref>.
-                </t>
-                <t>
-                    The simplest kind of link has an "href" with no template variables, and no "submissionSchema".  This does not
-                    allow for any variance in the link URI, nor does it allow for a request document.
-                </t>
-                <t>
-                    An "href" with at least one template variable, but no "hrefSchema" or "submissionSchema", allows resolving
-                    the template variable from the instance, but does not allow resolving it
-                    from external data, nor does it allow a request document.
-                </t>
-                <t>
-                    An "href" with at least one template variable and with an "hrefSchema" allows using external
-                    data to resolve the template, and falls back to resolving any remaining variables from the instance.
+                    <cref>
+                        Note that while the current draft does not provide a way to explicity
+                        indicate HTTP method support, some way of providing a non-authoritative
+                        hint may be added in a future draft (see issue #73 in the GitHub repository).
+                    </cref>
                 </t>
                 <t>
-                    A link with a "submissionSchema" allows submitting data for processing.
+                    Link Description Objects do not directly indicate what operations, such
+                    as HTTP methods, are supported by the target resource.  Instead, operations
+                    should be inferred primarily from link <xref target="rel">relation types</xref>
+                    and URI schemes.  Note, however, that a resource may always decline an operation at
+                    runtime, for instance due to application state that controls the operation's
+                    availability.
                 </t>
                 <t>
-                    See the individual keyword descriptions below for details related to each of these cases.
+                    There are several ways that a client can use data can with a link:
+                    <list>
+                        <t> URI Template variables resolved from server-supplied instance data </t>
+                        <t> URI Template variables resolved from client data </t>
```

Comment:
These two lines might be too easy to conflate. I would say something like

* Plain links, where all variables in the URI Template (if any) are filled from instance data
* Link templates, where the URI Template contains unbound variables to be filled in by users